components/automate-ui/src/app/helpers/auth/sorter.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { ChefSorters } from './sorter';
+
+describe('naturalSort', () => {
```

Comment:
I moved a whole slew of tests over from various components that now use ChefSorter so that the same style of test is being run in the sorter.spec file.  I didn't feel comfortable removing the similar tests from the other files, because many of those were using ChefSorter _inside_ of another function and it seemed like a good idea that those functions are still tested in the same manner.

https://github.com/chef/automate/blob/693c7f1be31d9df6bdbe46ba1c2a369e9af20983/components/automate-ui/src/app/pages/api-token/list/api-token-list.component.spec.ts#L96-L119

Reply:
On your concern about being inside other functions: good observation! But that was simply necessary boilerplate needed in order to get to test the thing we wanted to test. 

I think it would be beneficial to remove all those tests for a few reasons.
1. It was actually a code smell that all of those identical tests were propagated throughout the code base. Now, you've got them consolidated in a single place.
2. Testing this function that is now external to those other classes is somewhat outside the purview of those other classes to test, in the spirit of unit testing.
3. What if down the road we modify sorter.ts? One would then update sorter.spec.ts, but then one would likely need to update tests in all those other classes, too.

Reply:
That makes sense 👍 thanks for the explanation.  I've gone ahead and removed any tests only testing the sorting ability.